package-overrides/github/cruikshj/knockout-select2@1.0.0.json

```diff
@@ -1,7 +1,7 @@
 {
     "main": "dist/knockout-select2",
     "dependencies": {
-        "jquery": "npm:jquery@^2.2.0",
-        "knockout": "github:knockout/knockout@^3.3.0"
```

Comment:
It does not specifically depends on these versions, so it's better not to specify ?